pkg/filter/kafka/spec.go

```diff
@@ -21,5 +21,13 @@ type (
 	// Spec is spec of Kafka
 	Spec struct {
 		Backend []string `yaml:"backend" jsonschema:"required,uniqueItems=true"`
+		KVMap   *KVMap   `yaml:"mqtt" jsonschema:"required"`
+	}
+
+	// KVMap defines ways to get kafka data from MQTTContext kv map
+	KVMap struct {
+		Topic   string `yaml:"topic" jsonschema:"required"`
+		Headers string `yaml:"headers" jsonschema:"required"`
+		Payload string `yaml:"payload" jsonschema:"required"`
```

Comment:
Default payload has no sense, remove it

Reply:
If not use `Payload`, then we only can send data to kafka backend by using publish packet. For example, some mqtt client disconnect, and i want to send some data to kafak, then i can use MQTTContext.SetKV("payload", "some data") and use MQTTContext.GetKV("payload") to get it. `Payload` here is the key to call GetKV...

Reply:
Sorry for my misunderstood. The implementation is OK